repository/a.json

```diff
@@ -2306,6 +2306,18 @@
 				}
 			]
 		},
+		{
+			"name": "AutoPEP8_2020",
+			"details": "https://github.com/thep0y/SublimeAutoPEP8",
+			"labels": ["linting", "formatting"],
+			"readme": "https://raw.githubusercontent.com/thep0y/SublimeAutoPEP8/master/Readme.md",
```

Comment:
Since your "details" point to a GitHub repo, and the readme has a normal name and is in the root of the repo, the "readme" key not need.

Reply:
I will delete the "readme" key.